content/recipes/authoring/translation.md

```diff
@@ -7,4 +7,48 @@ next: /recipes/authoring/kumite/
+
+
+## Description
+
+- **Respect guidelines for [Writing a Kata Description](/recipes/authoring/kata-snippets/description/).**
+- **Use [Codewars Markdown Extensions](/references/markdown/extensions/#sequential-code-blocks)** to organize language specific parts of the description, if present. Add sections specific to your language if necessary.
+- If possible, **avoid displaying many language variants to the user**. Use conditional paragraphs to display information revelant to the currently viewed language.
+- If the language version can pass tests successfuly only when run with some specific language runtime, it should be specified in the description.
+
+
+## Code components
+
+- **Respect the [General Coding Guidelines](/recipes/authoring/kata-snippets/coding-general/), any conventions imposed by your [language](/languages/) and guidelines specific to each component of the kata**, e.g. the [preloaded](/recipes/authoring/kata-snippets/preloaded/) section, [reference solution](/recipes/authoring/kata-snippets/reference-solution/), [sample tests](/recipes/authoring/kata-snippets/sample-tests/) and [submission tests](/recipes/authoring/kata-snippets/full-tests/).
+- If possible, **sample tests and fixed tests should remain consistent between translations.** It may not always be feasible due to inherent differences in programming languages, but doing so can save follow-up questions on the language a user is attempting the kata in when a question on failing tests is raised in the kata discourse.
+- Sometimes, the description is not the best place to put some language specific information. For example, the translator may wish to add some information on arguments, data formats, memory management, etc. and keep the description language-agnostic at the same time. In such cases, it's allowed to put such information as an explanatory comment in the solution setup.
+
+
+## Review process
+
+- **The translation must be reviewed before it's approved.** If the reviewer is not familiar with the language of the translation, they should ask other users for help.
+- **Any issues found during review should be posted** as translation comments.
+- Unfortunately, the approval of a translation cannot be easily blocked by raising issues. If a translation has many issues, cannot be easily fixed, and there's justified concern that it could be approved prematurely while not keeping up to quality standards, **it should be rejected**.
+- After a translation is approved, it becomes **the responsibility of all parties involved: the user who approved the translation, the translator, and the kata author** (in this order) to fix any potential issues which might come up later when users attempt to submit their solutions.
```

Comment:
I disagree on the order, in this one.
I understand the reason (showing the great responsability of the approver), but the first responsible should imo be the translator himself. If the pressure is on the guy who didn't write the code, that doesn't sound reasonnable to me. And if it's officially documented that way:
* some morons (translators) will hide behind that
* if the kata author trusted the translator while he doesn't know the language, he cannot actually do anything so the order is biased
* if it actually turns that way and the approver actually has to handle the mess, very few translations will be aproved in the near future...

Reply:
Sounds very reasonable, but I wanted to make approves aware that they are not just pushing some green buttons for fun.

Reply:
Yup, I got that. It's effectively hard to state this clearly.
Maybe it's just not the appropriate place? Like, keeping here focus on the translator, and in the "approving/reviewing translations" doc, make strong evidence about the responsibility of the approver/reviewer?

Reply:
I think I will change the order as you suggest, and remove the strong note about the order.

Reply:
```suggestion
- After a translation is approved, it becomes **the responsibility of all parties involved: the translator, the user who approved the translation, and the kata author**, to fix any potential issues which might come up later when users attempt to submit their solutions.
```

Reply:
:+1: